editor/animation_track_editor.cpp

```diff
@@ -4107,6 +4107,12 @@ PropertyInfo AnimationTrackEditor::_find_hint_for_track(int p_idx, NodePath &r_b
 		property_info_base = property_info_base.get_named(leftover_path[i], valid);
 	}
 
+	if (property_info_base.is_null()) {
+		WARN_PRINT(vformat(RTR("Could not determine track hint for '%s' because its base property is null."),
```

Comment:
This is editor code, so it should use TTR, not RTR.
But actually, we don't translate error and warning prints, only editor strings, so this shouldn't use TTR.

Reply:
I'll try and revise it. I was a bit confused by this since there were other instances in the editor did this, so I wasn't sure what the rule actually was.